spring-boot-project/spring-boot-test/src/test/java/org/springframework/boot/test/context/SpringBootContextLoaderTests.java

```diff
@@ -88,18 +91,49 @@ public void environmentPropertiesNewLineInValue() {
 		assertKey(config, "variables", "foo=FOO\n bar=BAR");
 	}
 
+	@Test
+	public void noActiveProfiles() {
+		Environment environment = getApplicationEnvironment(SimpleConfig.class);
+		assertThat(environment.getActiveProfiles()).isEmpty();
+	}
+
+	@Test
+	public void multipleActiveProfiles() {
+		Environment environment = getApplicationEnvironment(MultipleActiveProfiles.class);
+		assertProfilesContains(environment, "profile1", "profile2");
```

Comment:
I'd find this easier to read as `assertThat(environment.getActiveProfiles()).containsExactly("profile1,2");` rather than calling the method, but that might just be me and I have no idea what the rest of the team might prefer.

Reply:
Same

Reply:
made it so